### ISSUE TITLE
Development of Teal functionality

### ISSUE THREAD

**Issue by leicray** [gear-genomics/teal#2]
Teal is a fantastic trace viewer but it currently lacks some key features that would make it ideal for use in a teaching environment. I do recognise that Teal was developed as a component of an analysis pipeline, but it has amazing potential for use in other ways.

- Would it be possible to display the quality scores as visually as vertical bars as well as showing the numerical values?
- It would be useful to scroll the trace horizontally using the mouse to drag the trace left and right, rather than using the buttons above the trace.
- Could the trace display  be linked to the chromatogram sequence such that a peak (or peaks) could be selected in the trace and the corresponding sequence be highlighted in the sequence box?

Many thanks.

**Comment by kaldari** [comment]
I agree it would be really nice to see the phred quality scores for each peak, either as a numerical value or visual element like a bar.

**Comment by naupaka** [comment]
Adding a +1 to this request. This is already an amazing tool for teaching in an undergrad context, and the enhancements @leicray mentions would make it pretty close to perfect.

Our use case is students get their trace files, look at the chromatogram, select out the good portion of the sequence, and then take that and BLAST it via NCBI's website.
